.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 require: rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 2.6
+  TargetRubyVersion: 2.7
```

Comment:
Shouldn't this be a 3.x version if that is what we are targeting?

Reply:
Correct!

Reply:
Actually, we were both wrong. This should be the oldest supported version of Ruby.